### PR TITLE
Minor fixes in `plasmapy.formulary` from PyCharm & Sourcery

### PR DESCRIPTION
**Pull request by namurphy**
These are a few quick minor automated fixes from using PyCharm's inspect code feature along with Sourcery.

### CODE REVIEW

File: plasmapy/formulary/magnetostatics.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def magnetic_field(self, p) -> u.T:
         )
 
         B_unit = np.cross(p2_p1, p - pf)
-        B_unit = B_unit / np.linalg.norm(B_unit)
+        B_unit /= np.linalg.norm(B_unit)
```
Comment (namurphy) [inline plasmapy/formulary/magnetostatics.py:277]:
```suggestion
        B_unit = B_unit / np.linalg.norm(B_unit)
```

File: plasmapy/formulary/magnetostatics.py [hunk 2]
```diff
@@ -415,8 +415,8 @@ def __init__(self, normal, center: u.m, radius: u.m, current: u.A, n=300):
             axis_x = np.array([1, 0, 0])
             axis_y = np.array([0, 1, 0])
         else:
-            axis_x = axis_x / np.linalg.norm(axis_x)
-            axis_y = axis_y / np.linalg.norm(axis_y)
+            axis_x /= np.linalg.norm(axis_x)
+            axis_y /= np.linalg.norm(axis_y)
```
Comment (namurphy) [inline plasmapy/formulary/magnetostatics.py:419]:
```suggestion
            axis_x = axis_x / np.linalg.norm(axis_x)
            axis_y /= axis_y / np.linalg.norm(axis_y)
```

File: plasmapy/formulary/mathematics.py [hunk 3]
```diff
@@ -148,14 +148,14 @@ def rot_a_to_b(a: np.ndarray, b: np.ndarray) -> np.ndarray:
         raise ValueError(
             f"Argument 'a' must have shape (3,) but input has shape {a.shape}."
         )
-    a = a / np.linalg.norm(a)
+    a /= np.linalg.norm(a)
```
Comment (namurphy) [inline plasmapy/formulary/mathematics.py:151]:
```suggestion
    a = a / np.linalg.norm(a)
```

**Review by StanczakDominik — APPROVED** [review]
:+1:

**Comment by namurphy** [comment]
Many thanks for the quick review!

**Pull request merged**